pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/optimizations/xml/UseStringBufferForStringAppends.xml

```diff
@@ -160,6 +160,7 @@ public class UseStringBuffer {
     public void foo2() {
         String country = request.getParameter("country");
         country = (country == null) ? "USA" : country;
+        country = (country != null) ? country : "USA";
```

Comment:
I'd rather have a new test case for this scenario, as seen they are independent (2 separate issues were reported and fixed on their own).